### DISCUSSION TITLE
implementation for high dimension dataset

### DISCUSSION THREAD

**Discussion by elhamum** [FlorentF9/DeepTemporalClustering#23] [Q&A]
Hello,
I am trying to use this network for data set of 5567-time series with 39792 columns on colab pro- Although I am able to run the network for sample datasets UCR but after the first epoch on my own dataset I got out of memory error. Any advice? I believe the network will take care of dimension  reduction so I am confused that what I should do

**Comment by sajinpayandath** [comment]
You can try reducing the number of filters n_units or Pool_size 10 or higher
